src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  let activeTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  if (!activeTab) {
+    [activeTab] = tabs;
+  }
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={classNames(
+                { 'is-active': tab.id === activeTab.id },
+              )}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (tab.id !== activeTab.id) {
+                    onTabSelected(tab);
+                    activeTab = tab;
+                  }
+                }}
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here